docs/src/content/en/docs/getting-started/installation.mdx

```diff
@@ -95,8 +99,45 @@ OPENAI_API_KEY=<your-api-key>
 ```
 > This example uses OpenAI. Each LLM provider uses a unique name. See [Model Capabilities](/docs/getting-started/model-capability) for more information.
 
+### Launch the Mastra Development Server
```

Comment:
The steps kinda left you hanging. You installed something but weren't told explicitly to start the dev server.